compiler/one-cmds/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ foreach(ONE_COMMAND IN ITEMS ${ONE_COMMAND_FILES})
                       GROUP_READ GROUP_EXECUTE
                       WORLD_READ WORLD_EXECUTE
           DESTINATION bin)
-          
+
```

Comment:
please remove this change. better to put in another PR.

Reply:
Ah. This diff is by vscode's setting. I'll fix it manually.